Zend/zend_ast.c

```diff
@@ -762,13 +768,6 @@ ZEND_API zend_result ZEND_FASTCALL zend_ast_evaluate(zval *result, zend_ast *ast
 				: NULL;
 
 			zend_class_entry *ce = zend_lookup_class(class_name);
-			if (!ce) {
```

Comment:
I'm not 100% sure if this can be removed

Reply:
You could always add ``ZEND_ASSERT(ce);`` which would crash if such a case exists